python/pyspark/pandas/tests/frame/test_reindexing.py

```diff
@@ -854,7 +879,8 @@ def test_sample(self):
 
 
 class FrameReidexingTests(FrameReindexingMixin, ComparisonTestBase, SQLTestUtils):
-    pass
+    def test_between_time(self):
+        super().test_between_time()
```

Comment:
We don't need to add a test here, since `FrameReidexingTests` already inherits `FrameReindexingMixin`.

Reply:
Removed

Reply:
> My bad, committed that accidentally, it was used for running only that one on local env

No worries, it also sometimes happen to me :-)